### PR TITLE
publish istio-1.6.5 release notes

### PR DESCRIPTION
**Pull request by irisdingbj**
publish istio-1.6.5 release notes.

### CODE REVIEW

File: content/en/news/releases/1.6.x/announcing-1.6.5/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+This release contains bug fixes to improve robustness. This release note describes
+what’s different between Istio 1.6.5 and Istio 1.6.4.
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** support to sort the pods by `creationTimestamp` during mixer lookup and return the latest pod.
```
Comment (jacob-delgado) [inline content/en/news/releases/1.6.x/announcing-1.6.5/index.md:19]: **Fixed** mixer lookup to always return the proper source name when performing lookups by IP when multiple pods had the same IP.

File: content/en/news/releases/1.6.x/announcing-1.6.5/index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- **Improved** Support to control sidecar injection based on revision at a per-pod level ([Issue 24801](https://github.com/istio/istio/issues/24801)
+- **Improved** `istioctl validate` will disallow unknown field from spec ([Issue 24860](https://github.com/istio/istio/issues/24860))
+- **Fixed** changed `stsPort` to `sts_port` in envoy bootstrap file. 
+- **Fixed** keep existing wasm state schema since it can be referenced by state objects later.
+- **Improved** added targetUri in stackdriver_grpc_service.
+- **Improved** updated wasm state to log for Access Log Service.
+- **Updated** Raised default protocol detection timeout from 100 Milliseconds to 5 seconds ([Issue 24379](https://github.com/istio/istio/issues/24379))
+- **Updated** removed UDP port 53 from Istiod service.
```
Comment (jacob-delgado) [inline content/en/news/releases/1.6.x/announcing-1.6.5/index.md:27]: **Removed** UDP port from 53 from Istiod service.

File: content/en/news/releases/1.6.x/announcing-1.6.5/index.md (hunk 1 above)
Comment (irisdingbj) [reply content/en/news/releases/1.6.x/announcing-1.6.5/index.md:19]: return the proper source name when mixer performing lookups by IP when multiple pods had the same IP.

File: content/en/news/releases/1.6.x/announcing-1.6.5/index.md (hunk 2 above)
Comment (irisdingbj) [reply content/en/news/releases/1.6.x/announcing-1.6.5/index.md:27]: make all using lower case now to align previous release notes

File: content/en/news/releases/1.6.x/announcing-1.6.5/index.md (hunk 1 above)
Comment (irisdingbj) [reply content/en/news/releases/1.6.x/announcing-1.6.5/index.md]: return the proper source name when mixer performing lookup by IP when multiple pods had the same IP.

File: content/en/news/releases/1.6.x/announcing-1.6.5/index.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+- **Updated** raised default protocol detection timeout from 100 Milliseconds to 5 seconds ([Issue 24379](https://github.com/istio/istio/issues/24379))
+- **Updated** removed UDP port 53 from Istiod service.
+- **Fixed** allow setting `status.sidecar.istio.io/port` to zero ([Issue 24722](https://github.com/istio/istio/issues/24722))
+- **Fixed**  support eds endpoint selection for subsets with no or empty label selector. ([Issue 24969](https://github.com/istio/istio/issues/24969))
+- **Fixed** fix wrong error log in operator.
+- **Fixed** allow `k8s.overlays` on `BaseComponentSpec`. ([Issue 24476](https://github.com/istio/istio/issues/24476))
+- **Fixed** istio-agent to create `eliptical` curve CSRs when `ECC_SIGNATURE_ALGORITHM` is set.
+- **Improved** map grpc status codes into http domain for metrics.
+- **Fixed** fix `scaleTargetRef` naming in `HorizontalPodAutoscaler` for istiod ([Issue 24809](https://github.com/istio/istio/issues/24809))
+- **Improved** added a version label to `pilot_xds` to give more information on data plane versions.
```
Comment (jacob-delgado) [inline content/en/news/releases/1.6.x/announcing-1.6.5/index.md:35]: This did not make the release. Can you remove it?

**Comment by frankbu** [comment]
@irisdingbj please use the work-in-progress label until the PR is ready to merge.

**Comment by duderino** [comment]
Hi @irisdingbj given the timezone difference and the need to get this done quickly, I took over this PR in https://github.com/istio/istio.io/pull/7692.

**Pull request closed without merging**